conf/messages.en

```diff
@@ -380,8 +380,8 @@ declaration.additionalActors.remove.empty = Select yes if you need to remove thi
 declaration.additionalActors.remove.title = Are you sure you want to remove this party?
 
 
-declaration.eori.empty = Enter an EORI in the correct format. For example, GB123456789000.
-declaration.eori.error.format = Enter an EORI in the correct format. For example, GB123456789000.
+declaration.eori.empty = Enter an EORI number in the correct format. For example, GB123456789000.
```

Comment:
should it be "For example: " or this form is also correct ?

Reply:
This is exactly how it was in the spreadsheet. I'll ask JW about it

Reply:
I am not saying it is wrong :) just being curious

Reply:
Confirmed: it should be comma.